app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -107,7 +104,7 @@ class MainFragment : Fragment() {
         }
         currentlyActiveProfile()
         setHasOptionsMenu(true)
-        return binding.root
+        return view
```

Comment:
one more change, use binding.root instead of the view. 
What is the use of creating a variable is not being used only once?
Make the change in other files also.

Reply:
@rob729 Ok, I also thought this at first, but I tried to follow the docs that is why. I will make the change.